samcli/commands/build/command.py

```diff
@@ -222,6 +229,7 @@ def do_cli(  # pylint: disable=too-many-locals, too-many-statements
     container_env_var_file: Optional[str],
     build_image: Optional[Tuple[str]],
     exclude: Optional[Tuple[str, ...]],
+    hook_package_id: Optional[str],
```

Comment:
why is the type hint different here than in the `cli` function? (shouldn't it be optional in `cli`?)

Reply:
good catch, it should be optional there too. will update